data/avro-schema/src/main/kotlin/net/corda/data/chunking/PropertyKeys.kt

```diff
@@ -6,4 +6,12 @@ object PropertyKeys {
      * Boolean property specifying whether same data content can re-upload again
      */
     const val FORCE_UPLOAD = "forceUpload"
-}
\ No newline at end of file
+    /**
+     * Boolean property specifying whether the vnode databases should be deleted and recreated
+     */
+    const val RESET_DB = "resetDb"
+    /**
+     * String property specifying the RPC actor that made the request
+     */
+    const val ACTOR = "actor"
```

Comment:
Ok, but seems strange to have actor as a property rather than as a String on these APIs seeing as it seems to be mandatory - we seem to always set it in forceCpiUpload and uploadCpi, and CpiValidatorImpl has a validation check to make sure its set. 

It would require changing a lot of places to do this so happy enough to go with this approach using property instead.